internal/handler/job/status/update.go

```diff
@@ -84,11 +87,18 @@ func Update(ctx context.Context, storage persistence.Storage, jobID string, newS
 		log.Error().Err(err).Msg("Failed to persist job update")
 		return nil, fault.Wrap(err)
 	}
+	contextLogger.Debug().Msg("Updated job status")
```

Comment:
In the other handlers, this information is logged after having published the event. Wouldn't it make sense to do it here as well?